docs/examples/github-actions.md

```diff
@@ -1,33 +1,40 @@
 # GitHub Actions
 
-The example below will run the testing-suite inside [github actions] (https://github.com/features/actions)
-for both PHP 7.4 and 8.1.
+The example below will run the testing-suite inside [github actions](https://github.com/features/actions)
+for PHP versions 8.1 through 8.4 and specify whether PHPUnit should also be run depending on PHP version.
 
 ```yml
 name: Testing Suite
 on: [push]
 jobs:
   PHP:
     strategy:
-      # Test with multiple PHP versions  
       matrix:
-        image: [
-          'srcoder/development-php:php74-fpm',
-          'srcoder/development-php:php81-fpm'
-        ]
+        php-version: [8.1, 8.2, 8.3, 8.4]
```

Comment:
Suggest to have

```
matrix:
    php:
        - { version: 8.1, tasks: composer,jsonlint,xmllint,yamllint,phpcs,phplint,phpmd,phpstan,securitychecker_enlightn }
        - and so on
```

And then you can use `matrix.php.tasks` in your script

Reply:
I will do this in a seperate PR / change so I can test the builds before opening a PR in case revision is needed